nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/domain/CustomTokenClient.java

```diff
@@ -115,7 +116,7 @@ public Map<String, String> getSubstitutableTokens() throws DomainException {
                             // Find next available unused port by incrementing the port value by 1
                             while (!NetUtils.isPortFree(port) && !usedPorts.contains(port)) {
                                 if (port > NetUtils.MAX_PORT) {
-                                    throw new DomainException("No free port available or it is prohibited.");
+                                    throw new DomainException(format("No free port available in range {0} - {1} or it is prohibited.", firstPortTried, port));
```

Comment:
However I think it would be better to use both borders inclusive, so the second parameter in the message would be `NetUtils.MAX_PORT`